ui/packages/catalog-ui-search/src/main/webapp/react-component/theme-settings/theme-settings.tsx

```diff
@@ -76,7 +76,12 @@ const ThemeCustom = styled.div`
 `
 
 const saveFontChanges = (fontSize: string) => {
-  getPreferences().set('fontSize', ThemeUtils.getFontSize(fontSize))
+  getPreferences().set(
+    'fontSize',
+    ThemeUtils.getFontSize(
+      ThemeUtils.getCalculatedZoomScale(fontSize, 1, 100, 62, 200)
```

Comment:
I noticed you `floored` the value from `getCalculatedZoomScale` above, do you need to do that here too?